### PR TITLE
Add a default footer for clubs.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: coss/templates/pages/menus/footer.html [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
   <div class="icon-container">
     <div class="icon">
-      <img class="footer-icon" src="/static/img/{{ link.icon }}.svg">
-      <a href="{{ link.target }}" target="_blank">{{ link.text }}</a>
+      <img class="footer-icon" src="/static/img/github.svg">
+      <a href="{{ settings.GITHUB_LINK }}" target="_blank">Contribute</a>
     </div>
   </div>
-{% endfor %}
+ {% endif %}
```
Comment (comzeradd) [inline coss/templates/pages/menus/footer.html]: I think you missed indentation here.

File: coss/templates/pages/menus/footer.html [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+
+{% if settings.LEGAL_LINK %}
+  <div class="icon-container">
+    <div class="icon">
+      <img class="footer-icon" src="/static/img/legal.svg">
+      <a href="{{ settings.LEGAL_LINK }}" target="_blank">Legal</a>
+    </div>
+  </div>
+ {% endif %}
```
Comment (comzeradd) [inline coss/templates/pages/menus/footer.html]: Same here.

File: coss/templates/pages/menus/footer.html [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
+
+{% if settings.LICENSE_LINK %}
+  <div class="icon-container">
+    <div class="icon">
+      <img class="footer-icon" src="/static/img/privacy.svg">
+      <a href="{{ settings.LICENSE_LINK }}" target="_blank">License</a>
+    </div>
+  </div>
+ {% endif %}
```
Comment (comzeradd) [inline coss/templates/pages/menus/footer.html]: Same here.

**Comment by akatsoulas** [comment]
@comzeradd pr updated

**Comment by a-kilroy** [comment]
@comzeradd do you want to add your footer suggested to the icebox so it's captured?

**Comment by comzeradd** [comment]
I opened an issue to keep track of that - #149

**Pull request merged**